packages/js/src/Modules/Verto/webrtc/BaseCall.ts

```diff
@@ -202,7 +202,29 @@ export default abstract class BaseCall implements IWebRTCCall {
     }
   }
 
-  hangup(params: any = {}, execute: boolean = true) {
+  /**
+   * Hangs up the call.
+   *
+   * @examples
+   *
+   * ```js
+   * call.hangup()
+   * ```
+   */
+  hangup(): void;
+  /**
+   * @internal
+   */
+  hangup(hangupParams, hangupExecute): void;
+  /**
+   * @internal
+   * @param params _For internal use_ Specify custom hangup cause and call ID
+   * @param execute _For internal use_ Allow or prevent execution of `Bye`
```

Comment:
There doesn't appear to be a way to omit specific params with Typedoc, even with Typescript function overloading. The closest I could get to an decent render is by specifying `@internal` and rendering it along with the first `hangup()` typing--using `@ignore` or `@hidden` hides `hangup` altogether.

This is how it looks rendered:
<img width="761" alt="Screen Shot 2020-12-04 at 1 48 56 PM" src="https://user-images.githubusercontent.com/4672952/101218359-c8820f00-3637-11eb-8bf9-21e618de1c37.png">

This way we could omit `@internal` methods when rendering in Developer Docs or other custom templates; any other ideas?